gui/wms_settings.html

```diff
@@ -1,15 +1,14 @@
 <div class="row">
-    <div class="col-5">
+    <div class="col-7 offset-5">
         <label class="checklabel">
-            <input type="checkbox" id="set_enable_interpolation" checked />
+            <input type="checkbox" data-callback="simpleWmsBodies.setEnableInterpolation" checked />
```

Comment:
Does that mean spatial interpolation for the texture lookups (GL_LINEAR, GL_NEAREST) or interpolating in time?

Reply:
It it time interpolation.